src/16F740.c

```diff
@@ -104,28 +115,39 @@ void btl_merlee_on_first_strike(void) {
 
                 if (currentEncounter->currentEnemy != NULL) {
                     if (currentEncounter->currentEnemy->flags & ACTOR_FLAG_NO_HEALTH_BAR) {
+                        // 46/101 ≈ 45.5%
                         if (temp <= 45) {
                             playerData->merleeSpellType = MERLEE_SPELL_1;
+                        // 45/101 ≈ 44.6%
                         } else if (temp <= 90) {
                             playerData->merleeSpellType = MERLEE_SPELL_2;
+                        // 10/101 ≈ 9.9%
                         } else {
                             playerData->merleeSpellType = MERLEE_SPELL_EXP_BOOST;
                         }
+                    // 31/101 ≈ 30.7%
                     } else if (temp <= 30) {
```

Comment:
The formatting here is putting me off slightly since technically this in the `if` block above, so it should be indented, but then that wouldn't look like it applied to the `else if`. I think `clang-format` would complain too if we ran it.

Perhaps we could do something like this:

```
 } else if (temp <= 30) { // 31/101 ≈ 30.7%
```
or
```
 } else if (temp <= 30 /* 31/101 ≈ 30.7% */) {
```

Does anyone else have any thoughts? I like both ways I suggested.

Reply:
This need only apply to the `else if` cases in this function, nowhere else in the PR IMO.

Reply:
I much prefer the first way. I have a strong aversion to comments in between code

Reply:
What about just having it as the first line in the codeblock itself? so 
``` c
 } else if (temp <= 30) {
     // 31/101 ≈ 30.7%
     ...
 }
 ```

Reply:
I think I prefer the aside `} else if (temp <= 30) { // 31/101 ≈ 30.7%`, but failing that, your suggestion marijn